src/modules/helpchan.ts

```diff
@@ -321,6 +335,27 @@ export class HelpChanModule extends Module {
 		}
 	}
 
+	private async updateStatusEmbed(channel: TextChannel, embed: MessageEmbed) {
+		let lastMessage = channel.messages.cache
+			.array()
+			.reverse()
+			.find(m => m.author.id === this.client.user?.id);
+
+		if (!lastMessage)
+			lastMessage = (await channel.messages.fetch({ limit: 5 }))
+				.array()
+				.reverse()
+				.find(m => m.author.id === this.client.user?.id);
```

Comment:
Add a filter so that playground links as the first message in a help channel don't cause a race condition and possibly get replaced

Reply:
I think it would make sense to look for specifically these embeds directly, something like:
```ts
const embed_titles = [ this.AVAILABLE_EMBED.title, /* … etc */ ];
let lastMessage = channel.messages.cache
    .array()
    .reverse()
    .filter(m => m.author.id === this.client.user?.id)
    .find(m => m.embeds.some(e => embed_titles.includes(e.title)))
```

Reply:
Copied from commit:

This detects status embeds by comparing their titles. In order to do that, it adds a title to the dormant embed. This isn't *necessary*; the following code would work instead:

```ts
const isStatusEmbed = (embed: MessageEmbed) => (
    embed.title === this.AVAILABLE_EMBED.title ||
    embed.title === this.OCCUPIED_EMBED_BASE.title ||
    embed.description === this.DORMANT_EMBED.description
);
```

However, this way is both cleaner code-wise, and I think it's nice for all the status embeds to have the same format.